src/nock.utils.test.ts

```diff
@@ -698,16 +834,127 @@ describe('nock utils', () => {
         });
       });
     });
+
+    describe('when sensitive information are URL encoded', () => {
+      it('replaces api key in body', async () => {
```

Comment:
```suggestion
      it('replaces parameter in body', async () => {
```

Here we are in integration test section.